src/repo/zcl_abapgit_repo.clas.abap

```diff
@@ -312,8 +311,6 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
 
     DATA: li_package TYPE REF TO zif_abapgit_sap_package.
 
-    find_remote_dot_abapgit( ).
-
     check_write_protect( ).
     check_language( ).
```

Comment:
I guess `check_language` uses the remote dot abap setting

however perhaps check_language() should also be removed, objects can be deleted no matter the current logon language?

Reply:
I will do another test, but `check_language` needs the remote setting only if the repo is brand new. Otherwise the stored dot settings are used. 

I think there was a reason to use the correct language. TADIR popups maybe

Reply:
okay 👍

Reply:
works